### PR TITLE
Fix lintint issues

### PR DESCRIPTION
**Pull request by xg-wang**
format, prettier code
add tslint task to scripts

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
     "watch": "broccoli-timepiece exports",
     "build": "./scripts/build.sh",
     "stats": "node scripts/size-calc",
+    "lint": "tslint src/**.ts",
```
Comment (asakusuma) [inline package.json]: Should we run this command on test? To make sure all commits are checked automatically?

Comment (xg-wang) [reply package.json]: added prettier🤘

**Comment by asakusuma** [comment]
Nice, TIL `husky` and `lint-staged`

File: src/metal/queue.ts [hunk 2]
```diff
@@ -9,19 +9,15 @@ distributed under the License is distributed on an "AS IS" BASIS,
...
+import { QueueInterface, QueueElementInterface, QueueDOMElementInterface } from './interfaces';
 
 export abstract class BaseQueue implements QueueInterface {
   protected items: Array<any>;
   constructor() {
     this.items = [];
   }
 
-  abstract removePredicate(identifier: any, element: any): void;
+  abstract removePredicate(identifier: any, element: any): boolean;
```
Comment (xg-wang) [inline src/metal/queue.ts]: This is the only non-format change

**Pull request merged**